### PR TITLE
Update dependencies and add github CI action

### PR DESCRIPTION
**Pull request by SergioMorchon**
All checks are passing, and the output in the `dist` folder is the expected ones (compared using git diff)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,17 @@
     "main": "./dist/webview-bridge-cjs.js",
     "module": "./dist/webview-bridge.mjs",
     "types": "./dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
```
Comment (SergioMorchon) [inline package.json:10]:
```
PS C:\Users\smorchon\dev\webview-bridge> npm pack --dry-run
npm notice
npm notice package: @tef-novum/webview-bridge@2.14.2
npm notice === Tarball Contents ===
npm notice 1.1kB  LICENSE
npm notice 4.6kB  dist/webview-bridge-cjs.js.flow
npm notice 27.0kB dist/webview-bridge-amd.js
npm notice 24.3kB dist/webview-bridge-cjs.js
npm notice 27.0kB dist/webview-bridge-iife.js
npm notice 8.9kB  dist/webview-bridge-iife.min.js
npm notice 27.3kB dist/webview-bridge-umd.js
npm notice 9.1kB  dist/webview-bridge-umd.min.js
npm notice 2.0kB  package.json
npm notice 2.5kB  CHANGELOG.md
npm notice 15.0kB README.md
npm notice 23.4kB dist/webview-bridge.mjs
npm notice 1.2kB  dist/src/analytics.d.ts
npm notice 166B   dist/src/calendar.d.ts
npm notice 685B   dist/src/contacts.d.ts
npm notice 475B   dist/src/device.d.ts
npm notice 652B   dist/src/dialogs.d.ts
npm notice 831B   dist/index.d.ts
npm notice 43B    dist/src/message-id.d.ts
npm notice 5.7kB  dist/src/post-message.d.ts
npm notice 649B   dist/src/sessions.d.ts
npm notice 1.9kB  dist/src/utils.d.ts
npm notice === Tarball Details ===
npm notice name:          @tef-novum/webview-bridge
npm notice version:       2.14.2
npm notice filename:      tef-novum-webview-bridge-2.14.2.tgz
npm notice package size:  34.9 kB
npm notice unpacked size: 184.7 kB
npm notice shasum:        1dcdfa780b65c718bd97780b3687c7d532adf862
npm notice integrity:     sha512-xzgjqn3qB1Pxz[...]/Oet9ijNCrLgw==
npm notice total files:   22
npm notice
tef-novum-webview-bridge-2.14.2.tgz
```

File: package.json [hunk 2]
```diff
@@ -24,19 +30,24 @@
...
+        "rollup": "^2.7.2",
+        "rollup-plugin-typescript2": "^0.27.0",
+        "ts-jest": "^25.4.0",
+        "typescript": "^3.8.3",
         "uglify-es": "^3.3.9"
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "npm run fix-code"
+        }
```
[the remark this answers is not in the document]
Comment (atabel) [reply package.json:51]: no, this is wrong, we need lint-staged, both tools work together.
lint-staged is needed to only apply the fixes to staged files
see recommended config by prettier: https://prettier.io/docs/en/precommit.html#option-1-lint-stagedhttpsgithubcomokonetlint-staged

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: CI
```
Comment (atabel) [inline .github/workflows/ci.yml:1]: Nice ;)

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (SergioMorchon) [reply package.json]: Oh I see, I've restored that

File: .github/workflows/ci.yml (hunk 3 above)
Comment (SergioMorchon) [reply .github/workflows/ci.yml:1]: Thank you :)

File: package.json (hunk 1 above)
Comment (pladaria) [reply package.json:10]: remove .npmignore

File: package.json [hunk 4]
```diff
@@ -24,19 +30,24 @@
...
     "devDependencies": {
         "@types/google.analytics": "0.0.40",
-        "@types/jest": "^24.0.18",
-        "@typescript-eslint/eslint-plugin": "^2.1.0",
-        "@typescript-eslint/parser": "^2.1.0",
-        "del-cli": "^2.0.0",
-        "eslint": "^6.3.0",
-        "husky": "^3.0.5",
-        "jest": "^24.9.0",
-        "lint-staged": "^9.2.5",
```
[the remark this answers is not in the document]
Comment (pladaria) [reply package.json:34]: husky sets the git hook, lint-staged executes stuff over staged files

[the remark this answers is not in the document]
Comment (SergioMorchon) [reply package.json:34]: thanks for the clarification!

File: package.json (hunk 1 above)
Comment (SergioMorchon) [reply package.json:10]: Yep, already removed

**Pull request merged**